spec/operators/publishReplay-spec.ts

```diff
@@ -358,34 +358,4 @@ describe('Observable.prototype.publishReplay', () => {
 
     published.connect();
   });
-
-  it('should follow the RxJS 4 behavior and NOT allow you to reconnect by subscribing again', (done: MochaDone) => {
-    const expected = [1, 2, 3, 4];
-    let i = 0;
-
-    const source = Observable.of(1, 2, 3, 4).publishReplay(1);
-
-    const results = [];
-
-    source.subscribe(
-      (x: number) => {
-        expect(x).to.equal(expected[i++]);
-      }, (x) => {
-        done(new Error('should not be called'));
-      }, () => {
-        source.subscribe((x: number) => {
-          results.push(x);
-        }, (x) => {
-          done(new Error('should not be called'));
-        }, () => {
-          done();
-        });
-
-        expect(() => source.connect()).to.throw(Rx.ObjectUnsubscribedError);
-      });
-
-    source.connect();
-
-    expect(results).to.deep.equal([4]);
-  });

```

Comment:
This was testing an inaccurate behavior. Rx4 didn't throw an error, it just didn't emit anything afterwards.